client/wfprev-war/src/main/angular/package-lock.json

```diff
@@ -9,12 +9,12 @@
       "version": "0.0.0",
       "dependencies": {
         "@angular/animations": "18.2.3",
-        "@angular/cdk": "18.2.3",
+        "@angular/cdk": "^18.2.3",
```

Comment:
We should be pinning our exact versions and not using these wildcards

Reply:
done